src/Microsoft.Identity.Web.AgentIdentities/Microsoft.Identity.Web.AgentIdentities.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+
+    <Title>Microsoft Identity Web Agentic Identity support</Title>
+    <Product>Microsoft Identity Web for Agent Identities</Product>
+    <Description>Helper methods for Agent applications to act as the agent identities.</Description>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+
+    <PackageValidationBaselineVersion>3.9.4</PackageValidationBaselineVersion>
```

Comment:
Would this be removed after this release? Or moved to Directory.Buiild.Props?

Reply:
this is a new package. so it's baseline will be it first version (for instance 3.10.0). For the moment we can't do version comparison as it's not released